### ISSUE TITLE
An integration with fluid systems.

### ISSUE THREAD

**Issue by TrueBlueFlare** [AlexModGuy/Ice-and-Fire-Suggestions#1311]
It'd be cool if dragon blood counted as a fluid that could interact with fluid pipes from various mods, that and it'd be gloriously messed up to have several comically large fluid tanks of dragon blood in your base.

**Comment by CTH999** [comment]
yeah! You could set up some sort of dragon-blood farm! Of course, that would only make sense in a more dark-magic modpack...

but yeah, I love this concept!
